src/avt/Database/Formats/avtFileFormat.h

```diff
@@ -257,11 +257,11 @@ class DATABASE_API avtFileFormat
     // and MTXX the void versions.
     //
     virtual void          GetCycles(std::vector<int>&) { return; };
-    virtual int           GetCycle(void) { return INVALID_CYCLE; };
-    virtual int           GetCycle(int) { return INVALID_CYCLE; };
+    virtual int           GetCycle(void);
```

Comment:
This change is confusing me. It moves default implementations of 4 of 6 methods here from header file to source file but leaves 2 in the header file. The commit message says this is to fix linker issue on Windows. Why didn't `GetCycles()` and `GetTimes` also need this change? What was the linker issue this change fixes?

Reply:
IIRC, the symbols were duplicated around. Making them non-inline fixed the duplicate symbols. It might be that the `std::vector` overload just isn't used in ParaView's codepaths?